week-5/B-callbacks/exercise.js

```diff
@@ -46,6 +48,7 @@ function exercise2(result) {
 
 function exercise3(callback) {
   // Write your implementation here
+  functionThatCallsBack(callback);
```

Comment:
This isn't quite right. This code will set the inner text for the `#exercise3` element to "Hello from the function caller". It should say "Hello from the callback". How could you fix this? Hint: the `callback` parameter that you are provided with is a function that you can call yourself inside `exercise3()`.

I can see that the exercise was maybe confusing, so I'll try to improve the wording.

Reply:
Oh! I did it in a hard way! Yes, I have not read clearly.